src/components/AddActor.jsx

```diff
@@ -0,0 +1,45 @@
+import React, {useState} from 'react';
+
+const AddActor = ({actors, setActors}) => {
+  const [addActorError, setAddActorError] = useState('');
+  const [newActor, setNewActor] = useState('');
```

Comment:
Could we grab this on submit instead of keeping it in state and causing the component to re-render every time a character is typed?

Reply:
Good catch. New implementation grabs the form element value instead of storing the value in state.